src/tools/run_onnx/run_onnx.cpp

```diff
@@ -0,0 +1,144 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+// tool to import and run an ONNX model
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "../../../test/util/test_tools.hpp"
```

Comment:
I'm not a fan of this include.
Is that possible to remove all of these `..`?

Reply:
Done.